File: Week_1_Day_3.js
// Check if an array is a palindrome

function isPalindrome(arr) {
    for(var left=0; left < arr.length/2; left++) {
        var right = arr.length - 1 - left;
        if(arr[right] !== arr[left]){
            return "Not a palindrome";
        }
    }
    return "Pal-indrome!"
}

var result1 = isPalindrome( [1, 1, 2, 2, 1] );
console.log(result1);
    
var result2 = isPalindrome( [3, 2, 1, 1, 2, 3] );
console.log(result2)

File: Week_1_Day_3.py
# Check if an array is a palindrome

def isPalindrome(arr):
    left = 0
    right = len(arr) -1

    while left < len(arr)/2:
        if arr[left] != arr[right]:
            return False
        else:
            left += 1
            right -= 1

    return True

print(isPalindrome([0, 1, 1, 0]))
print(isPalindrome([0, 1, 2, 1, 0]))
print(isPalindrome([1]))
print(isPalindrome([]))
print(isPalindrome([0, 1, 2, 3, 1, 0]))